CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Enhancements
 
 - Remove use of deprecated `Maze.driver.reset` in BitBar Appium tests [606](https://github.com/bugsnag/maze-runner/pull/606)
+- Add method to allow pulling of files via appium client [617](https://github.com/bugsnag/maze-runner/pull/617)
```

Comment:
Do you really want this to go into v8?

Reply:
I'll rebase it, forgot about the v8/9 split over crimbo

Reply:
Done